runtime/src/main/java/org/corfudb/runtime/collections/Table.java

```diff
@@ -272,6 +273,28 @@ private void initializeCorfuTable(CorfuRuntime runtime) {
         this.corfuTable = builder.open();
     }
 
+    /**
+     * This is a callback that is placed into the root transaction's context on
+     * the thread local stack which will be invoked right after this transaction
+     * is deemed successful and has obtained a final sequence number to write.
+     */
+    @AllArgsConstructor
+    class QueueEntryAddressGetter implements TransactionalContext.PreCommitListener {
```

Comment:
this class is not needed anymore, right?

Reply:
yes i moved it in the latest patchset to a different location